### PR TITLE
Added apoc.convert.toGraph procedure

### PR DESCRIPTION
**Pull request by vga91**
Added apoc.convert.toGraph procedure in order to recreate an apoc.convert.toTree procedure as before this change: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/1842

See https://github.com/neo4j-contrib/neo4j-apoc-procedures/compare/4.2.0.1..4.2#diff-2e97617732d6d793ed88b037b7a5aa86ffb6a61747e9e266d424e2e938b2ebbcR145

---

Required here: https://trello.com/c/ONciGFV0/882-customer-feature-request-requesting-new-procedure-for-older-patched-out-apocconverttotree-functionality

### CODE REVIEW

File: docs/asciidoc/modules/ROOT/pages/overview/apoc.convert/index.adoc [hunk 1]
```diff
@@ -13,6 +13,11 @@ This file is generated by DocsTest, so don't change it!
...
 |label:procedure[]
 |label:apoc-core[]
+|xref::overview/apoc.convert/apoc.convert.toGraph.adoc[apoc.convert.toGraph icon:book[]]
+
+apoc.convert.toGraph([paths],[lowerCaseRels=true], [config]) - equivalent to apoc.convert.toTree but it does not perform sorting operations
+|label:procedure[]
+|label:apoc-core[]
 |xref::overview/apoc.convert/apoc.convert.toTree.adoc[apoc.convert.toTree icon:book[]]
 
 apoc.convert.toTree([paths],[lowerCaseRels=true], [config]) creates a stream of nested documents representing the at least one root of these paths
```
Comment (ncordon) [inline docs/asciidoc/modules/ROOT/pages/overview/apoc.convert/index.adoc:23]: These two explanations seem very cryptic

**Comment by ncordon** [comment]
Looks good, but let's wait until a PM decision on this before merging

**Comment by vga91** [comment]
[[Customer Feature Request] Requesting new procedure for older patched out `apoc.convert.toTree()` functionality](https://trello.com/c/ONciGFV0/882-customer-feature-request-requesting-new-procedure-for-older-patched-out-apocconverttotree-functionality)

**Comment by vga91** [comment]
[[S3][Ericsson - PLM] Traverse a path with specified outgoing relation, some associated nodes through specified outgoing relation from a "common" node is missing when this node is intersect between two paths](https://trello.com/c/wnlJrLwn/876-s3ericsson-plm-traverse-a-path-with-specified-outgoing-relation-some-associated-nodes-through-specified-outgoing-relation-from-a)

**Comment by ncordon** [comment]
Hey @vga91, PMs have confirmed we are refraining from doing this at the moment

**Pull request closed without merging**